modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase

```diff
@@ -42,7 +42,7 @@ definition {
 	@description = "LPS-164728 - Assert that the listing displays the following columns: Organization Name, Status, Country, Contact Name, Contact Phone, Contact Email."
 	@priority = 4
 	test ManagerCanViewColumnsOnListing {
-		task ("Given a manager user with a manager role") {
+		task ("Given a manager user with a manager role is logged") {
```

Comment:
Maybe change to"Given a manager user with a manager role is logged in"

Reply:
"Please change to Given a user with a manager role is logged"

Reply:
lets try to keep a pattern when calling the _persona_.

Reply:
Ok, thanks for the review, I updated the PR just now